### PR TITLE
Update buildpacks pipeline to v0.2

### PR DESCRIPTION
**Pull request by bradbeck**
Update the buildpacks pipeline definition to v0.2.

Switch to using the Tekton catalog versions of the buildpacks task and pipeline definitions.

Remove a work-around in `examples/buildpacks/buildpacks.cue` which was setting a value for `ENV_VARS`.

Signed-off-by: Brad Beck <bradley.beck@gmail.com>

### CODE REVIEW

**Review by rgreinho — APPROVED** [review]

**Review by rgreinho — APPROVED** [review]

File: examples/buildpacks/buildpacks.cue [hunk 1]
```diff
@@ -30,9 +30,6 @@ frsca: pipelineRun: "cache-image-pipelinerun-": spec: {
 	}, {
 		name:  "CACHE_IMAGE"
 		value: _CACHE_IMAGE
-	}, {
-		name: "ENV_VARS"
-		value: [""]
```
Comment (sudo-bmitch) [inline examples/buildpacks/buildpacks.cue:35]: @bradbeck just verifying this is intentional because we had just added it in #248

**Review by sudo-bmitch — COMMENTED** [review]
I'll give my +1 after hearing back on ENV_VARS. Everything ran for me.

File: examples/buildpacks/buildpacks.cue (hunk 1 above)
Comment (rgreinho) [reply examples/buildpacks/buildpacks.cue:35]: I believe that's the workaround removed by this PR.

Comment (bradbeck) [reply examples/buildpacks/buildpacks.cue:35]: @sudo-bmitch Yep, the updated buildpacks pipeline definition includes this same value as its default.

**Review by sudo-bmitch — APPROVED** [review]

**Pull request merged**